content/setting-up-tugboat/connect-with-your-provider.md

```diff
@@ -274,3 +275,103 @@ You'll see a dialog box explaining what happens if you disconnect a linked accou
 ![Dialog box explaining disconnecting a linked git provider account with arrow to "Disconnect" button](../../_images/press-disconnect-button.png)
 
 {{% /expand%}}
+
+## Authenticating using an API access token
+
+By default, Tugboat uses OAuth to authenticate to git hosting providers. However, there may be times when GitHub,
+GitLab, or Bitbucket organizations take too long to approve org-level OAuth applications. As a workaround, you can use a
```

Comment:
This makes it sound like GH, GL, or BB can take a long time to authenticate OAuth applications. That's not really the case. The problem we are working around is when an organization admin can't or won't approve Tugboat at the org-level. Without that approval, the repo selector in the web interface will not work.